### DISCUSSION TITLE
Known DNS Providers wrong DNS for Applied Privacy DNS

### DISCUSSION THREAD

**Discussion by citizen-17** [AdguardTeam/AdGuardHome#2980] [Q&A]
The settings for Applied Privacy DNS are outdated https://kb.adguard.com/en/general/dns-providers#applied-privacy-dns

DNS, IPv4 | 94.130.106.88 -> 146.255.56.98
DNS, IPv6: 2a01 | 4f8:c0c:83ed::1 -> 2a02:1b8:10:234::2
DNS-over-HTTPS | https://doh.applied-privacy.net/dns-query -> https://doh.applied-privacy.net/query
DNS-over-TLS | tls://dot1.applied-privacy.net -> no changes

**Comment by Aikatsui** [comment]
@citizen-17 You can update it. 
1. Fork https://github.com/AdguardTeam/AdguardKnowledgeBase 
2. Edit [01.general/17.dns-providers/docs.en.md](https://github.com/AdguardTeam/AdguardKnowledgeBase/blob/master/01.general/17.dns-providers/docs.en.md) and make pull request.

**Reply by citizen-17** [reply]
Do I have to update the "Add to AdGuard" link as well? I don't understand how this link works or what exactly it does. 🤔

**Reply by vbagirov** [reply]
@citizen-17 basically you put the respective `sdns` there as a hyperlink. You can create DNS stamps here: https://dnscrypt.info/stamps/

 Make sure to select the correct protocol for each of the addresses. For example, for `146.255.56.98` we select "Plain DNS", as it's not encrypted, and the tool gives us `sdns://AAcAAAAAAAAADTE0Ni4yNTUuNTYuOTg`.
